lib/json.rs

```diff
@@ -45,32 +45,46 @@ pub fn to_json_value<T: serde::Serialize>(x: T) -> ddlog_std::Result<JsonValue,
     res2std(serde_json::to_value(x).map(JsonValue::from))
 }
 
-pub struct ValueWrapper(serde_json::value::Value);
-
-impl serde::Serialize for ValueWrapper {
+impl serde::Serialize for JsonValue {
     fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
     where
         S: serde::Serializer,
     {
         if serializer.is_human_readable() {
-            self.0.serialize(serializer)
+            match self {
+                JsonValue::JsonNull => serializer.serialize_unit(),
+                JsonValue::JsonBool { b } => serializer.serialize_bool(*b),
+                JsonValue::JsonNumber { ref n } => val_from_num(n.clone()).serialize(serializer),
```

Comment:
We should try and avoid this clone

Reply:
This one should be cheap, it's just cloning a numeric value.